packages/docs/docs/docs/linking.md

```diff
@@ -1,127 +1,281 @@
 
-Now, let's link our project to the openzeppelin-eth package by running:
+Let's now create and initialize our new `TokenExchange` contract:
 
 ```console
-npx zos link openzeppelin-eth
+$ zos create
+✓ Compiled contracts with solc 0.5.9 (commit.e560f70d)
+? Pick a contract to instantiate: TokenExchange
+? Pick a network: development
+✓ Contract TokenExchange deployed
+? Do you want to call a function on the instance after creating it?: Yes
+? Select which function: initialize(_rate: uint256, _token: address)
+? _rate (uint256): 10
+? _token (address): 0x2612Af3A521c2df9EAF28422Ca335b04AdF3ac66
+Instance created at 0x26b4AFb60d6C903165150C6F0AA14F8016bE4aec
 ```
 
-This command will install the openzeppelin-eth contracts locally, which is
-needed to compile the contracts that import it; but it will also update the
-`zos.json` file with a reference to the linked package to make sure that when
-you deploy the project it will use the EVM package that already exists on the
-blockchain.
+<!-- CODE: We need to extend the zos transfer command to support erc20s -->
 
-The following commands will be familiar to you. Just as we did before, we have
-to add the contract to the project:
+Our exchange is almost ready! We only need to fund it, so it can send tokens to purchasers. Let's do that using the `send-tx` command, to transfer the full token balance from our own account to the exchange contract. Make sure to replace the recipient of the transfer with the `TokenExchange` address you got from the previous command.
 
 ```console
-npx zos add MyLinkedContract
+$ zos send-tx
+? Pick a network: development
+? Choose an instance: StandaloneERC20 at 0x2612Af3A521c2df9EAF28422Ca335b04AdF3ac66
+? Select which function: transfer(to: address, value: uint256)
+? to (address): 0x26b4AFb60d6C903165150C6F0AA14F8016bE4aec
+? value (uint256): 10e18
+Transaction successful: 0x5863c8a8e122fcda7c6234abc6e60fad3f5a8108a3f88e2d8a956b63dbc222c2
+Events emitted: 
+ - Transfer
+    from: 0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1, 
+    to: 0x26b4AFb60d6C903165150C6F0AA14F8016bE4aec, 
+    value: 10000000000000000000
 ```
 
-> If any of the following commands fail with an `A network name must be provided 
-to execute the requested action` error, it means our session has expired. 
-In that case, renew it by running the command `npx zos session --network local 
---from 0x1df62f291b2e969fb0849d99d9ce41e2f137006e --expires 3600` again.
+All set! We can start playing with our brand new token exchange.
 
-Now, let's push our changes to the blockchain:
+## Using our exchange
+
+Now that we have initialized our exchange contract initialized, and seeded it with funds, we can test it out by purchasing tokens. Recall from our contract that the purchase is made automatically when we send ETH to the contract, so let's try it by using `zos transfer`, sending funds to the `TokenExchange` instance address:
```

Comment:
"Recall from our contract that the purchase is made automatically when we send ETH to the contract"

What's the context here? recall from where? 

We are talking about what our TokenExchange contract suppose to do.. correct? 
If yes, then I find this line confusing